docs/v2.5/howtos/intercepts.md

```diff
@@ -12,20 +12,36 @@ Telepresence enables you to create intercepts to a target Kubernetes workload. O
 
 For a detailed walk-though on creating intercepts using our sample app, follow the [quick start guide](../../quick-start/demo-node/).
 
-
 ## Prerequisites
```

Comment:
@njayp, 

> Users have to be able to do it without navigating to other pages, so there’s minimal friction (so no links to prerequisites or other pages with steps).

Right now users still need to go to TP installed page to figure out what to do, make a decision about the command-line tool, and go to an external page to see how to install the command-line tool, which results in more friction. For this particular quick start, we were thinking the first steps on this page to be:
1. Install Telepresence (and commands to do so)
2. Install the K8s command-line tool (and commands to do so) _since this is a golden path, we could ignore OpenShift_

Any thoughts?

Reply:
>Install Telepresence instructions
This makes sense and will be easy to add, I'll do it now

> Install kubectl instructions
I think this is a bit too much, I dont think we should maintain install instruction for external tools.